nnpdf_data/nnpdf_data/commondata/CDF_Z0_1P96TEV/metadata.yaml

```diff
@@ -53,9 +53,5 @@ implemented_observables:
     - - CDFZRAP_NEW-CDF_ZRAP_MCgrid_500M_last_two_bins_combined
   data_uncertainties:
     - uncertainties_ZRAP.yaml
-  variants:
-    legacy:
-      data_uncertainties:
-      - uncertainties_legacy_ZRAP.yaml
```

Comment:
You need to leave the variant here for now ^^U
but just make it point to the same `- uncertainties_ZRAP.yaml`

Once all datasets are done we can remove the variant (but many of them will need to remain... pointing to old fktables which might have different names)